hoot-core/src/main/cpp/hoot/core/ops/BuildingPartMergeOp.cpp

```diff
@@ -420,83 +408,4 @@ std::shared_ptr<geos::geom::Geometry> BuildingPartMergeOp::_getGeometry(
   }
   return std::shared_ptr<geos::geom::Geometry>();
 }
-
-RelationPtr BuildingPartMergeOp::combineBuildingParts(const OsmMapPtr& map,
-                                                      std::vector<ElementPtr>& parts)
```

Comment:
Moved this logic primarily into `BuildingMerger`, since its used by both it and `BuildingPartMergeOp`. Logically makes more since to be outside of here, since the new combine logic works for both building and multipoly relations.